services/src/main/java/org/keycloak/events/log/JBossLoggingEventListenerProvider.java

```diff
@@ -70,6 +71,11 @@ public void onEvent(Event event) {
                 sb.append(event.getError());
             }
 
+            if(event.getDetails().containsKey(Details.CLIENT_SESSION_STATE))
+                event.getDetails().remove(Details.CLIENT_SESSION_STATE);
+            if(event.getDetails().containsKey(Details.CLIENT_SESSION_HOST))
+                event.getDetails().containsKey(Details.CLIENT_SESSION_HOST);
```

Comment:
@HabooshHaddad Don't you mean event.getDetails().**remove**(Details.CLIENT_SESSION_HOST);  ?

Reply:
Yes, you are right.